pkg/kernel/kernel.go

```diff
@@ -27,14 +26,15 @@ func GetKernelVersion() (*VersionInfo, error) {
 		flavor               string
 	)
 
-	_, stdout, _, err := exec.Run(0, "uname", "-r")
+	buf := unix.Utsname{}
+	err := unix.Uname(&buf)
```

Comment:
Can we add a benchmark test for the `Uname` interface?

Reply:
Done, plz check it.